tests/app/Services/JsonServiceTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
```

Comment:
[declare(strict_types=1);](suggestion: Add declare(strict_types=1); in new class and files)

Reply:
But good idea for test files